socorro/processor/rules/mozilla.py

```diff
@@ -54,16 +54,58 @@ class CopyFromRawCrashRule(Rule):
+        ("string", "AdapterDeviceID", "adapter_device_id"),
+        ("string", "AdapterDriverVersion", "adapter_driver_version"),
+        ("string", "AdapterSubsysID", "adapter_subsys_id"),
+        ("string", "AdapterVendorID", "adapter_vendor_id"),
+        ("string", "Android_Board", "android_board"),
+        ("string", "Android_Brand", "android_brand"),
+        ("string", "Android_CPU_ABI", "android_cpu_abi"),
+        ("string", "Android_CPU_ABI2", "android_cpu_abi2"),
+        ("string", "Android_Device", "android_device"),
+        ("string", "Android_Display", "android_display"),
+        ("string", "Android_Fingerprint", "android_fingerprint"),
+        ("string", "Android_Hardware", "android_hardware"),
+        ("string", "Android_Manufacturer", "android_manufacturer"),
+        ("string", "Android_Model", "android_model"),
+        ("string", "Android_Version", "android_version"),
+        ("string", "AppInitDLLs", "app_init_dlls"),
+        ("string", "AsyncShutdownTimeout", "async_shutdown_timeout"),
+        ("int", "AvailablePageFile", "available_page_file"),
+        ("int", "AvailablePhysicalMemory", "available_physical_memory"),
+        ("int", "AvailableVirtualMemory", "available_virtual_memory"),
+        ("string", "BIOS_Manufacturer", "bios_manufacturer"),
+        ("string", "CoMarshalInterfaceFailure", "co_marshal_interface_failure"),
+        ("int", "ContentSandboxCapabilities", "content_sandbox_capabilities"),
+        ("int", "ContentSandboxLevel", "content_sandbox_level"),
+        ("string", "CPUMicrocodeVersion", "cpu_microcode_version"),
+        ("string", "GMPLibraryPath", "gmp_library_path"),
+        ("string", "GraphicsCriticalError", "graphics_critical_error"),
+        ("int", "InstallTime", "install_time"),
+        ("string", "ipc_channel_error", "ipc_channel_error"),
+        ("string", "IPCFatalErrorMsg", "ipc_fatal_error_msg"),
+        ("string", "IPCFatalErrorProtocol", "ipc_fatal_error_protocol"),
+        ("string", "IPCMessageName", "ipc_message_name"),
+        ("int", "IPCMessageSize", "ipc_message_size"),
+        ("string", "IPCShutdownState", "ipc_shutdown_state"),
+        ("int", "IPCSystemError", "ipc_system_error"),
+        ("string", "MemoryErrorCorrection", "memory_error_correction"),
+        ("int", "OOMAllocationSize", "oom_allocation_size"),
+        ("string", "RemoteType", "remote_type"),
+        ("string", "ShutdownProgress", "shutdown_progress"),
+        ("int", "StartupTime", "startup_time"),
+        ("int", "SystemMemoryUsePercentage", "system_memory_use_percentage"),
+        ("int", "TotalPageFile", "total_page_file"),
+        ("int", "TotalPhysicalMemory", "total_physical_memory"),
+        ("int", "TotalVirtualMemory", "total_virtual_memory"),
+        ("float", "UptimeTS", "uptime_ts"),
+        ("string", "useragent_locale", "useragent_locale"),
+        ("string", "Vendor", "vendor"),
```

Comment:
This is not a great way of doing things, but it's a small step in the right direction.

At some point, we'll write a schema that specifies which fields are sourced from the raw crash and what their types are. Then this rule will use that to do its thing.